### PR TITLE
Fixing transcription errors in REENTRY_CONTROL.

### PR DESCRIPTION
**Pull request by abrogley**
Pages proofed: 844 to 882

Fixes chrislgarry/Apollo-11#250

### CODE REVIEW

File: Comanche055/REENTRY_CONTROL.agc [hunk 1]
```diff
@@ -1463,7 +1463,7 @@ VREFER		DEC	.019288		# REFERENCE VELOCITY SCALED V/51532.3946
 # DEFINED BY EQUALS
 
 DEC15		=	LOW4
-#GAMMAL1	=	22D
+GAMMAL1		=	22D
```
Comment (abrogley) [inline Comanche055/REENTRY_CONTROL.agc]: In split diff view, the spacing looks wrong, but in [full file](https://github.com/abrogley/Apollo-11/blob/84cb4f66477952e360974d800decc6b0841fcacf/Comanche055/REENTRY_CONTROL.agc#L1466) view it looks okay

File: Comanche055/REENTRY_CONTROL.agc [hunk 2]
```diff
@@ -54,8 +54,8 @@ STARTENT	EXIT			# MM = 63
 		INHINT
 		MASK	CM/FLAGS
 					# ENTRYDSP = 92D B13
-					# GONEPAST=95D B10	RELVELSW=96D B9
-					# EGSW = 97D B8
+					# GONEPAST=95D B10.	RELVELSW=96D B9
+					# EGSW = 97D B8         NOSWITCH = 98D B7
```
Comment (oldmud0) [inline Comanche055/REENTRY_CONTROL.agc]: This line requires tabs instead of spaces to match line 57.

File: Comanche055/REENTRY_CONTROL.agc (hunk 1 above)
Comment (oldmud0) [reply Comanche055/REENTRY_CONTROL.agc]: Must be the leading `+` that is messing it up.

File: Comanche055/REENTRY_CONTROL.agc (hunk 2 above)
Comment (abrogley) [reply Comanche055/REENTRY_CONTROL.agc]: Changed to tabs.

**Pull request merged**